### DISCUSSION TITLE
part-ii/2017-114/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#41] [Question comments]
Part II 2017 Principles of Quantum Mechanics: Paper 3, Section II, C 

https://questions.tripos.org/part-ii/2017-114/

**Comment by silkca** [comment]
The main content of the question is just an algebra bash using the commutation relations given.

For the second half, you need to bear in mind the analogy with the angular momentum operators - $\left(a^{\dagger}\right)^{2}|0\rangle$ has $K_{3}$-eigenvalue 1, and $K^{2}$-eigenvalue 2, and for the $J$ operators we have $|j,m_j\rangle$ states such that $J_{3}|j,m_j\rangle=m_j|j,m_j\rangle$ and $J^2|j,m_j\rangle=j(j+1)|j,m_j\rangle$, with allowed values $j,m_j\in \mathbb{Z}$, $-j\leq m_j\leq j$, and raising and lowering operators $J_{\pm}$ such that $J_{\pm}|j,m_j\rangle\propto |j,m_j\pm 1\rangle$

If any of this is confusing then read back your notes on angular momentum replacing each $J$ with $K$.
